vispy/app/backends/_pyqt4.py

```diff
@@ -13,7 +13,7 @@
 
 try:
     # Make sure no conflicting libraries have been imported.
-    for lib in ['PySide', 'PyQt5', 'PySide2', 'PySide6']:
+    for lib in ['PySide', 'PyQt5', 'PyQt6', 'PySide2', 'PySide6']:
         lib += '.QtCore'
         if lib in sys.modules:
             raise RuntimeError("Refusing to import PyQt4 because %s is "
```

Comment:
This code is repeated in each Qt backend and can surely be refactored away. This could be fixed here, but maybe its easier if one of us (e.g. me) does this once this PR has merged.